src/Core/src/Handlers/Border/BorderHandler.iOS.cs

```diff
@@ -51,8 +48,24 @@ static partial void UpdateContent(IBorderHandler handler)
 			if (handler.VirtualView.PresentedContent is IView content)
 			{
 				var platformContent = content.ToPlatform(handler.MauiContext);
-				platformContent.Tag = ContentView.ContentTag;
-				platformView.AddSubview(platformContent);
+
+				// If the content is a UIScrollView, we need a container to handle masks and clip shapes effectively.
+				if (platformContent is UIScrollView)
```

Comment:
Could we create an UITest and verify the Border after update editors using https://github.com/maonaoda/TestMaui/tree/bug/25817?